### PR TITLE
UI Updates

### PR DESCRIPTION
**Pull request by haihan-lin**
- [x] Move filter to a tab on the left tool box instead of the drawer
- [x] Update all icon, use trashcan icon for clear all charts, replay icon for reset filter.
- [x] Update About modal, adding team names and more links.
- [x] Show little x for extra pair plot only on hover
- [x] Consistent UI for adding chart element: add an insert chart icon for add button on top, and change the icon in chart to blue that will add elements.
- [x] Add a "mode" for showing patient detail and ids.

### CODE REVIEW

**Comment by JackWilb** [comment]
I'm seeing this issue with the blood components when I load in. The filter range is set to 0-0 instead of the whole range
image

**Comment by JackWilb** [comment]
It's possible to get into a broken state where you have case info showing with no cases in the selector above by brushing and then clicking the "x" in currently selected

image

**Comment by haihan-lin** [comment]
is this after the data being loaded?

**Comment by haihan-lin** [comment]
> It's possible to get into a broken state where you have case info showing with no cases in the selector above by brushing and then clicking the "x" in currently selected
> 
> image

I'm a bit confused

**Comment by JackWilb** [comment]
> is this after the data being loaded?

This is just when loading into the application for the first time. So yes, after all the data has loaded

**Comment by JackWilb** [comment]
> I'm a bit confused

Maybe this will help show what I mean
https://user-images.githubusercontent.com/36867477/135891279-f3cd0e14-8af0-4993-bd72-61c599ea2756.mp4

File: frontend/src/Components/Modals/SaveStateModal.tsx [hunk 1]
```diff
@@ -20,33 +20,33 @@ const SaveStateModal: FC = () => {
...
+                if (response.status === 200) {
+                    onSuccess(false);
+                } else {
+                    response.text().then(error => {
+                        onFail(response.statusText);
+                    })
+                }
+            }).catch(error => {
+                onFail(error.toString())
+            })
```
Comment (JackWilb) [inline frontend/src/Components/Modals/SaveStateModal.tsx:49]: This looks good, but what if we want to open a state, modify it, and then save it as a new one?

File: frontend/src/Components/Utilities/DetailView/CaseInfo.tsx [hunk 2]
```diff
@@ -18,8 +18,9 @@ const CaseInfo: FC = () => {
     const generate_List_Items = () => {
         let result = [];
         if (individualInfo) {
+            console.log(individualInfo)
```
Comment (JackWilb) [inline frontend/src/Components/Utilities/DetailView/CaseInfo.tsx:21]: Is this going to console log the HIPAA sensitive data?

File: frontend/src/Components/Utilities/FilterInterface/FilterBoard.tsx [hunk 3]
```diff
@@ -66,206 +65,196 @@ const FilterBoard: FC = () => {
...
+                                    } else {
+                                        setEndDate(rawDateRange[1])
+                                    }
+                                }} />
+                        </MuiPickersUtilsProvider>} />
+                </ListItem>
+                {/* </List>
+
+
+        <List dense> */}
```
Comment (JackWilb) [inline frontend/src/Components/Utilities/FilterInterface/FilterBoard.tsx:130]: Can we remove this comment? And there's a few below, too

**Review by JackWilb — CHANGES_REQUESTED** [review]
Looks great overall. I think once we fix up those small issues I already commented on, this PR is ready to go. Thanks for the hard work!

File: frontend/src/Components/Modals/SaveStateModal.tsx (hunk 1 above)
Comment (haihan-lin) [reply frontend/src/Components/Modals/SaveStateModal.tsx:49]: I am addressing everything with states in the new PR (haven't started by on my todo list this week.)

**Comment by haihan-lin** [comment]
> > I'm a bit confused
> 
> Maybe this will help show what I mean https://user-images.githubusercontent.com/36867477/135891279-f3cd0e14-8af0-4993-bd72-61c599ea2756.mp4

broken link?

File: frontend/src/Components/Modals/SaveStateModal.tsx (hunk 1 above)
Comment (JackWilb) [reply frontend/src/Components/Modals/SaveStateModal.tsx:49]: #186

**Comment by haihan-lin** [comment]
the selection issue should be fixed. I also fixed the issue of scatterplot repeatedly having actions registered in provenance but not actually updating the selection (which caused the undo/redo not functioning properly)

**Review by JackWilb — APPROVED** [review]
Looks good! I think the filter menu could maybe use some more whitespace and grouping, but it shouldn't slow this PR down. Thanks for pushing those fixes

**Pull request merged**